src/query/storages/fuse/src/operations/read_partitions.rs

```diff
@@ -91,7 +96,7 @@ impl FuseTable {
                         .await;
                 }
 
-                if (!dry_run && snapshot.segments.len() > nodes_num) || is_lazy {
+                if !is_inverted && ((!dry_run && snapshot.segments.len() > nodes_num) || is_lazy) {
```

Comment:
We don't support distributed read for inverted index?

Reply:
Another question here is:
Why here need to check `!is_inverted` but others like `aggregate index` no need?

Better to have more comments.